src/upload-result.js

```diff
@@ -13,6 +13,7 @@ governing permissions and limitations under the License.
 import { getAverage } from './utils';
 import HttpResult from './http-result';
 import UploadError from './upload-error';
+import CreateDirectoryResult from './create-directory-result';
```

Comment:
I see this is used in the JSDoc below. We could add an eslint ignore rule so that CI will pass (currently failing on a linting error: https://github.com/adobe/aem-upload/actions/runs/4671308102/jobs/8272122932)